lib/senders/templates/post_verify_secondary_email.html

```diff
@@ -1,78 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <title>{{t "Firefox Accounts"}}</title>
-</head>
```

Comment:
What's the story with this template getting removed entirely, was it here in error in the first place?

Reply:
This was introduce by accident way back when the template has a different name. It was created automatically with `grunt templates` and is safe to be deleted now.